src/table.js

```diff
@@ -26,18 +26,18 @@ function Table(props) {
         props.onClickIcon(e.target.parentElement.parentElement.id, props.id);
     }
 
-    const isGNDVisionTable = () => {
-        if(props.table.categorytitle !== undefined && props.table.categorytitle === "Green New Deal vision") {
-            return true
-        }
-        else return false;
-    }
+    // If there's a category title, it is the GND Vision section
+    const isGNDVisionTable = (props.table.categorytitle) ? true : false;
+
+    // Check for 'How Much' table id so we can render it appropriately
+    const isHowMuchTable = (props.id === 1) ? true : false;
```

Comment:
🎨 might be better to rely on something "stable" like table-id rather than an index, in case it moves (also not a big deal)